### ISSUE TITLE
SAML Configuration for Tenant

### ISSUE THREAD

**Issue by Kennochas** [vmware/terraform-provider-vcd#1041]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

My organization is working on a big project to replace some custom API work that had been done, with terraform. A feature we require is the ability to add the ability to control SAML authentication for tenants. 

### New or Affected Resource(s)

vcd_org_saml

### Terraform Configuration (if it applies)

```hcl
Copy-paste your Terraform configurations here. Please **shrink HCL replicating the bug to minimum** 
without modules. It will help us to respond and replicate the problem quicker. It may also uncover
some HCL error while doing so.

For bigger configs you may use [Gist(s)](https://gist.github.com) or a service like Dropbox and
share a link to the ZIP file.
```

### References

https://docs.vmware.com/en/VMware-Cloud-Director/10.4/VMware-Cloud-Director-Tenant-Portal-Guide/GUID-1F1F3EFD-55C5-4BF7-8683-FD93184A402F.html

#565

**Comment by dataclouder** [comment]
@Kennochas,
This feature is implemented in PR #1064 
Testing it thoroughly, however, is not a simple matter.
Could you build a  provider using the branch of PR #1024 and see if it does what you expect?
Feel free to contact the team in Slack channel `#vcd-terraform-dev` ( in slack space vmwarecode.slack.com ) for feedback.

**Comment by dataclouder** [comment]
implemented in [v3.10.0](https://github.com/vmware/terraform-provider-vcd/releases/tag/v3.10.0)
